sqlglot/dialects/doris.py

```diff
@@ -58,6 +58,7 @@ class Generator(MySQL.Generator):
             exp.StrToUnix: lambda self, e: self.func("UNIX_TIMESTAMP", e.this, self.format_time(e)),
             exp.Split: rename_func("SPLIT_BY_STRING"),
             exp.TimeStrToDate: rename_func("TO_DATE"),
+            exp.AddMonths: rename_func("MONTHS_ADD"),
```

Comment:
You need to also map `MONTHS_ADD` to `exp.AddMonths` otherwise the Doris -> <other dialects> way won't work.